deploy/operator.yaml

```diff
@@ -15,14 +15,21 @@ spec:
       serviceAccountName: kie-cloud-operator
       containers:
         - name: kie-cloud-operator
-          # Replace this with the built image name
           image: quay.io/kiegroup/kie-cloud-operator
           ports:
             - containerPort: 60000
               name: metrics
           command:
             - kie-cloud-operator
           imagePullPolicy: Always
+          readinessProbe:
+            exec:
+              command:
+                - stat
+                - /tmp/operator-sdk-ready
+            initialDelaySeconds: 4
+            periodSeconds: 10
+            failureThreshold: 1
```

Comment:
Should we have a similar health probe?

Reply:
Any thoughts on liveness probe? Does OpenShift warn you when one is missing? Can the same readiness probe be used for liveness? Should it?

Reply:
can we think a little more about the liveness probe? i'm not sure this script is the best way to go about it -
https://github.com/operator-framework/operator-sdk/blob/dba3d18eec0bada10d13cbe7ba474df5f3e99321/pkg/ready/ready.go#L45

The readiness probe is a good start.

Reply:
it appears to just create the file and the readiness probe then ensures it exists ... not a great way to approach liveness